zstacklib/zstacklib/utils/lvm_range.py

```
@@ -0,0 +1,695 @@
+            _error("LVM_RANGE_PV_DEVICE_AMBIGUOUS",
+                   "PV[%s] resolves to[%s] block-device candidates" %
+                   (pv_name, len(matches)))
+
+        candidate, paths = matches[0]
+        paths.extend([pv_name, real_pv_name])
+        result.append(build_block_device_evidence(
+            candidate, paths, realpath, list_slaves, capacity_reader))
+
+    target_wwids = set(device["wwid"] for device in result)
+    candidate_count_by_wwid = {}
+    for candidate, unused_paths in candidates:
```

Ruff (0.16.2)

[warning] Loop control variable `unused_paths` not used within loop body

Rename unused `unused_paths` to `_unused_paths`

(B007)